test/TestApp/Program.cs

```diff
@@ -91,7 +91,6 @@ private static void Main()
                 InferenceEngine.DefaultEngine.Compiler.RecommendedQuality = QualityBand.Preview;
                 InferenceEngine.DefaultEngine.Compiler.GenerateInMemory = true;
                 InferenceEngine.DefaultEngine.Compiler.WriteSourceFiles = false;
-                TestUtils.TestAllCompilerOptions(true);
```

Comment:
There is a good reason why this was here.  It allows debugging into a test that failed during the compiler options build.  How does debugging a failure work in this new setup?

Reply:
That makes sense, thanks. I've re-enabled this now (and put the line back).